hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/conf/TestConfiguration.java

```diff
@@ -2557,11 +2546,11 @@ public void testGettingPropertiesWithPrefix() throws Exception {
     assertTrue(prefixedProps.isEmpty());
   }
 
-  public static void main(String[] argv) throws Exception {
+  /*public static void main(String[] argv) throws Exception {
```

Comment:
Could this not be translated to JUnit 5?

I'm actually not sure if it's valuable anyway. If it's not needed, let's delete it instead of commenting it out.